fluid/face_detection/image_util.py

```diff
@@ -177,6 +247,28 @@ def generate_batch_samples(batch_sampler, bbox_labels, image_width,
     return sampled_bbox
 
 
+def generate_batch_random_samples(batch_sampler, bbox_labels, image_width,
+                                  image_height, scale_array, resize_width,
+                                  resize_height):
+    sampled_bbox = []
+    index = []
+    c = 0
```

Comment:
Is index and c needed? seems never be used

Reply:
Removed.